src/hipscat/io/paths.py

```diff
@@ -67,9 +79,54 @@ def pixel_catalog_file(catalog_path="", pixel_order=0, pixel_number=0):
     norder = int(pixel_order)
     npix = int(pixel_number)
     ndir = int(npix / 10_000) * 10_000
-    return os.path.join(
-        catalog_path,
+    return append_paths_to_pointer(
+        catalog_base_dir,
         f"{ORDER_DIRECTORY_PREFIX}={norder}",
         f"{DIR_DIRECTORY_PREFIX}={ndir}",
         f"{PIXEL_DIRECTORY_PREFIX}={npix}.parquet",
     )
+
+
+def get_catalog_info_pointer(catalog_base_dir: FilePointer) -> FilePointer:
+    """Get file pointer to `catalog_info.json` metadata file
+
+    Args:
+        catalog_base_dir: pointer to base catalog directory
+    """
```

Comment:
Should this (and the functions below) also have the return value in the comments?

Reply:
Yep